treeherder/etl/perf_data_adapters.py

```diff
@@ -101,8 +95,7 @@ def _calculate_test_data(job_id, result_set_id, push_timestamp,
             "job_id": job_id,
             "result_set_id": result_set_id,
             "push_timestamp": push_timestamp,
-            "total_replicates": r_len,
```

Comment:
Do we not want to store the number of replicates the value was generated from? it ranges from 1 (v8_7) to dozens (dromaeo), and anything in between.